### PR TITLE
[Upgrade][Quincy][Cephadm]Adding upgrade scenarios

### PR DESCRIPTION
**Pull request by MohitBis**
[Upgrade][Quincy][Cephadm]Adding upgrade scenarios

Scenarios:
Upgrade from RHCS 5x to RHCS 6x with fix os
Upgrade from RHCS 5x to RHCS 6x in RHEL 9

### CODE REVIEW

**Comment by MohitBis** [comment]
Logs:
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-SJ3Q3H/
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-5JTUEY/

File: tests/cephadm/test_remove_labels_installer.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
+from cli.cephadm.cephadm import CephAdm
+from utility.log import Log
+
+log = Log(__name__)
```
Comment (vamahaja) [inline tests/cephadm/test_remove_labels_installer.py:4]: Unused variable

File: tests/cephadm/test_remove_labels_installer.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+
+class RemoveService(Exception):
```
Comment (vamahaja) [inline tests/cephadm/test_remove_labels_installer.py:7]: Chane name to `RemoveServiceError`

File: cli/ceph/orch/orch.py [hunk 3]
```diff
@@ -99,3 +99,16 @@ def ps(self, **kw):
         if isinstance(out, tuple):
             return out[0].strip()
         return out
+
+    def label_rm(self, hostname, labelname):
```
Comment (vamahaja) [inline cli/ceph/orch/orch.py:103]: this function should be moved here - https://github.com/red-hat-storage/cephci/blob/master/cli/ceph/orch/label.py

**Review by vamahaja — CHANGES_REQUESTED** [review]

File: tests/cephadm/test_remove_labels_installer.py (hunk 1 above)
Comment (MohitBis) [reply tests/cephadm/test_remove_labels_installer.py]: Thanks @vamahaja 
Updated

File: tests/cephadm/test_remove_labels_installer.py (hunk 2 above)
Comment (MohitBis) [reply tests/cephadm/test_remove_labels_installer.py]: Thanks @vamahaja
Updated

File: cli/ceph/orch/orch.py (hunk 3 above)
Comment (MohitBis) [reply cli/ceph/orch/orch.py]: Thanks @vamahaja
Updated

**Comment by MohitBis** [comment]
Latest Log:
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-VUD2H3/

File: suites/quincy/upgrades/tier-1_upgrade_5x-to-6x_mix_os.yaml [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+#--------------------------------------------------------------------------------
+# - Deploy RHCS 5 cluster with RHEL 8 and RHEL 9 hosts
+# - Add _admin label to RHEL 9 hosts
+# - Remove RHEL 8 _admin node, RHEL 9 _Admin node act as admin node
+# - Upgrade cluster from rhcs 5 to rhcs 6
+# - Validate cluster status
+# - Run I/O's
+#--------------------------------------------------------------------------------
+#################################################################################
+tests:
```
Comment (vamahaja) [inline suites/quincy/upgrades/tier-1_upgrade_5x-to-6x_mix_os.yaml:18]: ?????

File: cli/ceph/orch/label.py [hunk 5]
```diff
@@ -21,3 +21,16 @@ def add(self, node, label):
         if isinstance(out, tuple):
             return out[0].strip()
         return out
+
+    def label_rm(self, hostname, labelname):
```
Comment (vamahaja) [inline cli/ceph/orch/label.py:25]: Use name `rm`

File: suites/quincy/upgrades/tier-1_upgrade_5x-to-6x_mix_os.yaml (hunk 4 above)
Comment (MohitBis) [reply suites/quincy/upgrades/tier-1_upgrade_5x-to-6x_mix_os.yaml:14]: Thanks @vamahaja 
Updated

File: cli/ceph/orch/label.py (hunk 5 above)
Comment (MohitBis) [reply cli/ceph/orch/label.py]: Thanks @vamahaja 
Updated

File: cli/ceph/orch/label.py [hunk 6]
```diff
@@ -21,3 +21,16 @@ def add(self, node, label):
         if isinstance(out, tuple):
             return out[0].strip()
         return out
+
+    def rm(self, hostname, labelname):
```
Comment (vamahaja) [inline cli/ceph/orch/label.py:25]: change `labelname` to `label`

Comment (MohitBis) [reply cli/ceph/orch/label.py]: Thanks @vamahaja 
Updated

**Review by vamahaja — APPROVED** [review]

**Pull request merged**